js/core/devices.js

```diff
@@ -156,7 +156,7 @@ const uaParsers = {
  * @module core/devices
  * @export default
  */
-const Devices = Class.inherit({
```

Comment:
export class Devices... ?

Reply:
We don't use Devices as class in other components. We create `const devices = new Devices();` and export it.
There is no need to `export class Devices`, but we can do it.